collection_requirements.rst

```diff
@@ -110,7 +110,13 @@ Repository management
 Repository name
 ---------------
 
-For collections under ansible-collections the repo SHOULD be named ``NAMESPACE.COLLECTION``
+For collections under ansible-collections the repository SHOULD be named ``NAMESPACE.COLLECTION``.
```

Comment:
under ansible-collections -> from ansible-collections github project ?
Should be we talk about convention for corporate maintained collection (`COMPANY_NAME.PRODUCT` or `COMPANY_NAME.SERIES`)